src/components/Licenses/SBOMViewer.tsx

```diff
@@ -1,79 +1,96 @@
 };
-interface CycloneDXExternalRef {
-  url?: string;
-  type?: string;
-  comment?: string;
-}
 
-interface CycloneDXLicense {
-  license?: {
-    id?: string;
-  };
+interface GitHubPackage {
+  name: string;
+  SPDXID: string;
+  versionInfo?: string;
+  downloadLocation?: string;
+  filesAnalyzed?: boolean;
+  licenseConcluded?: string;
+  copyrightText?: string;
+  externalRefs?: {
+    referenceCategory?: string;
+    referenceType?: string;
+    referenceLocator?: string;
+  }[];
 }
 
-interface CycloneDXProperties {
-  name?: string;
-  value?: string;
-}
-
-interface CycloneDXComponent {
-  type?: string;
-  name?: string;
-  group?: string;
-  version?: string;
-  bomRef?: string;
-  author?: string;
-  description?: string;
-  licenses?: CycloneDXLicense[];
-  externalReferences?: CycloneDXExternalRef[];
-  properties?: CycloneDXProperties[];
-}
-
-interface CycloneDXTool {
-  name?: string;
-  version?: string;
-  vendor?: string;
-  externalReferences?: CycloneDXExternalRef[];
-}
-
-interface CycloneDXBOM {
-  bomFormat?: string;
-  specVersion?: string;
-  version?: number;
-  serialNumber?: string;
-  metadata?: {
-    timestamp?: string;
-    tools?: CycloneDXTool[];
-    component?: CycloneDXComponent;
+interface GitHubSBOM {
+  sbom?: {
+    spdxVersion?: string;
+    dataLicense?: string;
+    SPDXID?: string;
+    name?: string;
+    documentNamespace?: string;
+    creationInfo?: {
+      creators?: string[];
+      created?: string;
+    };
+    packages?: GitHubPackage[];
   };
-  components?: CycloneDXComponent[];
 }
 
 const BOMDisplay: React.FC = () => {
   const [copyStatus, setCopyStatus] = useState(false);
   const [showExternalRefs, setShowExternalRefs] = useState<number | null>(null);
   const [activeTab, setActiveTab] = useState<string>("bom");
+  const [feBomData, setFeBomData] = useState<GitHubSBOM | null>(null);
+  const [beBomData, setBeBomData] = useState<GitHubSBOM | null>(null);
+
+  useEffect(() => {
+    const fetchSBOMData = async (url: string): Promise<GitHubSBOM | null> => {
+      try {
+        const response = await fetch(url, {
+          headers: {
+            Accept: "application/vnd.github+json",
+            "X-GitHub-Api-Version": "2022-11-28",
+          },
+        });
+        return response.ok ? await response.json() : null;
+      } catch (error) {
+        console.error("Error fetching SBOM data:", error);
+        return null;
+      }
+    };
+
+    const fetchData = async () => {
+      const feUrl =
+        "https://api.github.com/repos/ohcnetwork/care_fe/dependency-graph/sbom";
+      const beUrl =
+        "https://api.github.com/repos/ohcnetwork/care/dependency-graph/sbom";
+
+      const [frontendData, backendData] = await Promise.all([
+        fetchSBOMData(feUrl),
+        fetchSBOMData(beUrl),
+      ]);
+
+      setFeBomData(frontendData);
+      setBeBomData(backendData);
+    };
+
+    fetchData();
+  }, []);
```

Comment:
You can use useQuery with fetch instead.

Also, instead of fetching dynamically everytime on runtime, i'd recommend adding this data-fetching logic to `postinstall` step or build step.

Refer npm scripts: https://docs.npmjs.com/cli/v9/using-npm/scripts#examples

During the step, fetch the data using gh's APIs and save it as a JSON (gitignore it).
Lazy import the JSON directly in this file, and render accordingly.

cc: @bodhish